governance/tag-governance.md

```diff
@@ -423,59 +415,55 @@ organizing and shepherding the execution of the subproject. They adhere to the
 requirements, duties and expectations outlined in the Subproject Lead role
 description in CNCF Community Group Leadership Requirements.
 
-Recommended minimum number of leads: 2  
-Term: 1 year  
+Recommended minimum number of leads: 2
+Term: 1 year
 Election requirements: Super-majority vote of the parent governing body.
 
-Subproject Leads are the technical authority for a subproject. They MUST have
-demonstrated both good judgement and responsibility towards the health of that
-subproject.
-
-
-##### Requirements
-
-- MUST be approved by a super-majority vote of the parent governing body.  
-- MUST adhere to the [Technical Leadership Principles].  
+- MUST adhere to the [Technical Leadership Principles].
 - MUST be nominated by the TAG’s Tech Leads (minimum 1) for TAG led subprojects,
   or by the TOC (minimum 1) for TOC led subprojects.
-- Membership MUST be tracked alongside the parent governing body metadata.  
+- Membership MUST be tracked alongside the parent governing body [metadata].
 - SHOULD have prior experience within the CNCF (e.g. as an active TAG community
   member), but MAY be approved by their parent governing body for an explicitly
   documented reason such as extensive domain expertise, formation of a new
-  governing body (e.g. new TAG or Working Group), or demonstrating significant
-  aptitude for people & community management. 
+  governing body (e.g. new TAG or subproject), or demonstrating significant
+  aptitude for people & community management.
 
 
 ##### Duties
 
 - SHOULD set direction and make or approve design decisions for their
-  subproject - either directly or through delegation.   
+  subproject - either directly or through delegation.
   - Design decisions that make changes outside of the subproject’s defined scope
-    MUST be approved by the Tech Lead(s) of the TAG.  
+    MUST be approved by the Tech Lead(s) of the TAG for TAG level subprojects,
+    or by the TOC for TOC level subprojects.
 - MUST actively participate in triaging issues and reviewing PRs related to
-  their subproject.  
-- MUST regularly report subproject status to TAG leadership.  
+  their subproject.
+- MUST regularly report subproject status to the parent governing body.
 - SHOULD be an escalation point for technical discussions and decisions in the
-  subproject.  
-- SHOULD set milestone priorities.   
+  subproject.
+- SHOULD set milestone priorities. Subprojects MAY have other roles, but are
+  REQUIRED to be documented in the subproject charter and adhere to the
+  requirements outlined in the Other Roles section of the CNCF Community Group
+  Leadership Requirements.
```

Comment:
Should this be two bullets? Or is this an erroneous copy+paste from the lower section?

```suggestion
- SHOULD set milestone priorities.
```

Reply:
sigh - definitely an accidental paste